circleci-orbs/preview.yml

```diff
@@ -0,0 +1,95 @@
+description: Easily register and deploy a component to a new environment on the Architect Cloud, then destroy the component and environment when ready
+
+display:
+  home_url: https://architect.io
+  source_url: https://github.com/architect-team/architect-cli/tree/master/circleci-orbs
+
+jobs:
+  create:
+    parameters:
+      email:
+        type: string
+      password:
+        type: string
+      account:
+        type: string
+      environment:
+        type: string
+      platform:
+        type: string
+      component_name:
+        type: string
+      component_file:
+        type: string
+        default: ./architect.yml
+      deploy_flags:
+        type: string
+        default: ''
+    executor: preview_executor
+    steps:
+      - run: npm install -g @architect-io/cli@0.7
+      - run: architect login -e << parameters.email >> -p << parameters.password >>
```

Comment:
use the `ARCHITECT_PASSWORD` env instead of explicitly setting -p since it will show up in logs. 

if we do this for the github action - can you create an issue to address there too?